docker-jans-cloudtools/scripts/certmanager.py

```
@@ -664,6 +693,25 @@ def prune(self):
+            new_jwks = json.loads(f.read()).get("keys", [])
+
+        # old JWK that is not listed in new JWKS will be saved as an archived JWK
+        new_jwks_kids = {new_jwk["kid"] for new_jwk in new_jwks}
+        for old_jwk in old_jwks:
+            if old_jwk["kid"] in new_jwks_kids:
+                continue
+
+            try:
+                self.backend.create_archived_jwk(old_jwk, lifetime)
+                logger.info("JWK %s has been archived", old_jwk["kid"])
+            except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)